c++/include/orc/Reader.hh

```diff
@@ -149,6 +149,25 @@ namespace orc {
      */
     RowReaderOptions& includeTypes(const std::list<uint64_t>& types);
 
+    /**
+     * A map type of <typeId, ReadIntent>.
+     */
+    typedef std::map<uint64_t, ReadIntent> TypeReadIntents;
+
+    /**
+     * Selects which type ids to read and specific ReadIntents for each
+     * type id. The root type is always 0 and the rest of the types are
```

Comment:
`root type is always 0` might sound weird. Technically, `root type ID is always 0`.
BTW, I'd like to recommend to remove this sentence, `The root type is always 0 and the rest of the types are labeled in a preorder traversal of the tree` because this is redundant.

Reply:
Removed the sentece.